src/metabase/models/permissions_group.clj

```diff
@@ -40,17 +40,26 @@
   "All Users")
 
 (def ^{:arglists '([])} all-users
-  "Fetch the `All Users` permissions group, creating it if needed."
+  "Fetch the `All Users` permissions group"
```

Comment:
This doesn't create the group if needed anymore -- that's done in a Liquibase migration now

Reply:
Yes, that's why I'm removing it from the comment

Reply:
Oh oops, I'm dumb. I got what was being added/removed reversed -- I thought things were getting reverted because you added the MetaBot group code back -- nvm